network/src/commonMain/kotlin/com/wire/kalium/network/api/keypackage/KeyPackageApiImpl.kt

```diff
@@ -4,18 +4,20 @@ import com.wire.kalium.network.AuthenticatedNetworkClient
 import com.wire.kalium.network.api.UserId
 import com.wire.kalium.network.utils.NetworkResponse
 import com.wire.kalium.network.utils.wrapKaliumResponse
-import io.ktor.client.HttpClient
 import io.ktor.client.request.get
+import io.ktor.client.request.parameter
 import io.ktor.client.request.post
 import io.ktor.client.request.setBody
 
 class KeyPackageApiImpl internal constructor(private val authenticatedNetworkClient: AuthenticatedNetworkClient) : KeyPackageApi {
 
     private val httpClient get() = authenticatedNetworkClient.httpClient
 
-    override suspend fun claimKeyPackages(user: UserId): NetworkResponse<ClaimedKeyPackageList> =
+    override suspend fun claimKeyPackages(user: UserId, selfClientId: String?): NetworkResponse<ClaimedKeyPackageList> =
```

Comment:
Is there a reason for selfClientId to be nullable?

Reply:
selfClientId is a queryParam, and also it's only needed when we are claiming keypackages for current account.

Reply:
So maybe we can add something to make it clear that passing the selfClientId will call the endpoint with `skip_own` quarry parameter.